java/fury-core/src/main/java/org/apache/fury/serializer/AbstractObjectSerializer.java

```diff
@@ -256,6 +306,29 @@ private InternalFieldInfo[] buildFieldsInfo() {
     return fieldInfos;
   }
 
+  public static InternalFieldInfo[] buildFieldsInfo(Fury fury, List<Field> fields) {
+    List<Descriptor> descriptors = new ArrayList<>();
+    for (Field field : fields) {
+      if (!Modifier.isStatic(field.getModifiers())) {
```

Comment:
transient fields should be filtered too

Reply:
We can just `fury.getClassResolver().getAllDescriptorsMap(beanClass, false).values();` instead